--- a/grid/Pegasus/Makefile.LINUX
+++ b/grid/Pegasus/Makefile.LINUX
@@ -26,8 +26,8 @@
 		mDiffFit mGridExec mNotify mPresentation
 
 mConcatFit:	mConcatFit.o
-		$(CC) -o mConcatFit mConcatFit.o \
-		../../Montage/checkFile.o $(LIBS)
+		$(CC) -o mConcatFit mConcatFit.o \
+		../../Montage/debugCheck.o ../../Montage/checkFile.o $(LIBS)
 
 mDAG:		mDAG.o hashtable.o
 		$(CC) -o mDAG mDAG.o hashtable.o $(LIBS)
@@ -40,11 +40,11 @@ mDAGGalacticPlane:		mDAGGalacticPlane.o hashtable.o
 		$(CC) -o mDAGGalacticPlane mDAGGalacticPlane.o hashtable.o $(LIBS)
 
 mDAGTbls:	mDAGTbls.o
-		$(CC) -o mDAGTbls mDAGTbls.o  \
+		$(CC) -o mDAGTbls mDAGTbls.o ../../Montage/debugCheck.o \
 		../../Montage/filePath.o  \
 		../../Montage/checkHdr.o  \
 		../../Montage/checkWCS.o $(LIBS)
 
 mDiffFit:	mDiffFit.o
-		$(CC) -o mDiffFit mDiffFit.o $(LIBS)
+		$(CC) -o mDiffFit mDiffFit.o ../../Montage/debugCheck.o $(LIBS)
 
 mGridExec:	mGridExec.o
 		$(CC) -o mGridExec mGridExec.o $(LIBS)
